src/doom/d_main.c

```diff
@@ -182,9 +189,18 @@ boolean D_Display (void)
     int				y;
     boolean			wipe;
     boolean			redrawsbar;
-		
+
+    if(!start_snoflakes && crispy->snowflakes)
+    {
+        for(size_t i = 0; i < 5000; i++)
```

Comment:
Does this survive zn in-game High Resolution or Widescreen Rendering switch?

Reply:
Yes 🤔 
it works nice for me